examples/mlperf/optimizers.py

```diff
@@ -0,0 +1,37 @@
+from typing import List, Set
+
+from tinygrad import Tensor
+from tinygrad.nn.optim import Optimizer
+
+# https://github.com/mlcommons/training/blob/master/image_classification/tensorflow2/lars_optimizer.py
+class LARS(Optimizer):
+  def __init__(self, params: List[Tensor], lr, momentum=0.9, weight_decay=1e-4, eta=0.001, eps=0.0, skip_list=None, nesterov=False):
+    super().__init__(params, lr)
+    self.momentum, self.weight_decay, self.eta, self.eps, self.nesterov = momentum, weight_decay, eta, eps, nesterov
+    self.b = [Tensor.zeros(*t.shape, device=t.device, requires_grad=False) for t in self.params]
+    self.skip_list = set(skip_list or [])
+
+  def step(self):
+    for i, t in enumerate(self.params):
+      assert t.grad is not None
+      g = t.grad.contiguous()
```

Comment:
why contiguous?

Reply:
In SGD, it realizes the grad because in earlier version of lazy, there was diamond problem. I think it should be fixed, but I don't want to make big changes here. So contiguous instead of realize because then it can go into one schedule.